lib/resty/lmdb/transaction.lua

```diff
@@ -71,6 +71,45 @@ function _M.begin(hint)
 end
 
 
+local normalize_key
+do
+    local sha256
+
+    local ok, digest = pcall(require, "resty.openssl.digest")
+    if not ok then
+
+        local resty_sha256 = assert(require("resty.sha256").new())
```

Comment:
I think we only need to use `lua-resty-string`, since it is shipped with all OpenResty installations. I don't see much advantage of supporting both `lua-resty-openssl` and `lua-resty-string` at the same time.

Reply:
In kong we only use `lua-resty-openssl`, it may be a better choice.